components/portal/messages/controllers.js

```diff
@@ -66,27 +66,36 @@ define(['angular'], function(angular) {
          */
         var filterMessages = function() {
           // Check if group filtering has been disabled
-          if (!$localStorage.disableGroupFilteringForMessages) {
+          if ($localStorage.disableGroupFilteringForMessages) {
             // Define promises to run if filtering is turned on
             promiseFilteredMessages = {
               filteredByGroup:
+                messagesService.getAllMessages(),
+              filteredByData:
+                messagesService.getMessagesByData(allMessages),
+              filteredByTitle:
+                messagesService.getMessagesByTitle(allMessages),
+            };
+          } else {
+            promiseFilteredMessages = {
+              filteredByGroup:
                 messagesService.getMessagesByGroup(allMessages),
               filteredByData:
                 messagesService.getMessagesByData(allMessages),
               filteredByTitle:
                 messagesService.getMessagesByTitle(allMessages),
             };
+          }
             // Call filtered notifications promises, then pass on to
             // the completion function
             $q.all(promiseFilteredMessages)
               .then(filterMessagesSuccess)
               .catch(filterMessagesFailure);
-          } else {
+         
```

Comment:
Looks like the CI builds are failing on this line have some extra spaces.
https://ci.appveyor.com/project/davidmsibley/uportal-app-framework/build/1.0.528